adaptived/src/utils/path_utils.c

```diff
@@ -269,3 +269,20 @@ API void adaptived_path_walk_end(struct adaptived_path_walk_handle **handle)
 
 	*handle = NULL;
 }
+
+API int adaptived_file_exists(const char * const path)
+{
+	char check_path[FILENAME_MAX];
+	char *subp;
+
+	strcpy(check_path, path);
+	subp = strstr(check_path, "*");
```

Comment:
I'm not a fan of replacing `*` with \0.  While unusual, `*` is valid in filenames and could occur anywhere in the file.

I don't think we want this check.